### PR TITLE
Added spanish translation, added missing link to Sprig slack channel

### PR DESCRIPTION
**Pull request by wollygfx**
Hey there, just added the spanish translation to this workshop, and added a missing link to the sprig slack channel

### CODE REVIEW

File: workshops/sprig/es-xl.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+Dígale a la gente que empiece por trabajar con el [tutorial](https://editor.sprig.hackclub.com/?file=https://raw.githubusercontent.com/hackclub/sprig/main/games/getting_started.js).
+
+Una vez que la gente haya completado el tutorial, anímala a que empiece a pensar en su propio juego. A menudo, una buena forma de crear un juego original es empezar con uno ya existente y añadirle una nueva mecánica. Echa un vistazo a los juegos etiquetados como "hackeables" para encontrar bases remezclables.
+
+## Compartir las creaciones (10 minutos)
+
+Llama la atención de todos y pídeles que compartan sus juegos contigo. Pregunta si alguien quiere hacer una demostración de su juego al grupo. Celebra el trabajo de todos y aprovecha para que cada persona reciba comentarios, inspiración y afirmación.
+
+Si los juegos están completos, entonces [hacer un Pull Request](https://sprig.hackclub.com/share) para presentarlos a la galería y [ganar una consola Sprig](https://sprig.hackclub.com).
```
Comment (javierzaleta) [inline workshops/sprig/es-xl.md:63]: I would suggest to change `hacer un Pull Request` with `puedes hacer un Pull Request`.

**Review by javierzaleta — CHANGES_REQUESTED** [review]

File: workshops/sprig/es-xl.md (hunk 1 above)
Comment (javierzaleta) [reply workshops/sprig/es-xl.md:63]: Everything else looks good!

Comment (wollygfx) [reply workshops/sprig/es-xl.md:63]: Ohh yes, i missed that

**Comment by wollygfx** [comment]
Hey @sampoder, why has this PR been ignored?

**Comment by sampoder** [comment]
I missed Javi's message, apologies. LGTM.

**Review by sampoder — APPROVED** [review]

**Pull request merged**